functions/public/scripts/profile.js

```diff
@@ -89,8 +89,7 @@ Profile.prototype.saveProfile = function (year, course, university) {
         message: 'Profile saved!',
         timeout: 2000
       };
-      //self.signInSnackbar.MaterialSnackbar.showSnackbar(data);
-      window.history.back();
+      document.getElementById("must-signin-snackbar").innerHTML = data;
```

Comment:
Why do we show the "must-signin-snackbar" here? I'm guessing this should be renamed to a generic name? It's really confusing. Maybe call it "snackbar"?